### ISSUE TITLE
Werid viewpoint orientation

### ISSUE THREAD

**Issue by szx0112** [ethz-asl/StructuralInspectionPlanner#9]
Hi,

I find this interesting work recently and implement the applications on a mesh model of terrain as below. I changed the parameters of sensor pitch angle to 45 because I think the UAV should be always above the terrian, and reduce the FOV of camera to simulate a real one. But based on the visualization result, I found that most viewpoints are nearly horizontal that I think is not very reasonable. 

Does the parameters I set are wrong? Any comment is appreciate. 

Thanks,

Holistic view:

![screenshot from 2017-07-22 19-16-58](https://user-images.githubusercontent.com/10392640/28495441-78ac7c94-6f12-11e7-9ba4-b6020ea13ca8.png)

Detailed view:

![screenshot from 2017-07-22 19-24-00](https://user-images.githubusercontent.com/10392640/28495474-58855804-6f13-11e7-902f-0568a71384e4.png)

**Comment by birchera** [comment]
Hi @szx0112,

The displayed poses, location and [heading](https://github.com/ethz-asl/StructuralInspectionPlanner/blob/master/koptplanner/src/plan.cpp#L544), correspond to the vehicle pose. The camera field of view would be relative to it, as you defined it, i.e. with the 45° pitch.

If I remember correctly I tried once to have the sensor direction displayed, but this makes it very hard to perceive on a 2D screen where the arrows point exactly in 3D. Therefore I went with the heading only.

Hope that helps.
